packages/dev/docs/pages/react-spectrum/ssr.mdx

```diff
@@ -103,6 +104,7 @@ const withTM = require("next-transpile-modules")([
   "@react-spectrum/well",
   "@spectrum-icons/ui",
   "@spectrum-icons/workflow"
+  "@spectrum-icons/workflow/node_modules/@react-spectrum/icon",
```

Comment:
Not sure what change made this need to be added, but I couldn't build without it.

Reply:
I think it can be removed now, https://github.com/adobe/react-spectrum/pull/2933

Reply:
It's still giving me issues without this on 3.16.4: 

> ./node_modules/@spectrum-icons/workflow/node_modules/@react-spectrum/icon/dist/main.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules/@spectrum-icons/workflow/node_modules/@react-spectrum/icon/dist/module.js

Reply:
can you determine why? did the workflow icon pinning update not go through with 3.16.4 and @spectrum-icons/workflow@3.2.4?